Marlin/src/lcd/ultralcd.cpp

```diff
@@ -1438,6 +1438,22 @@ void MarlinUI::update() {
 
   #endif
 
+  #if ENABLED(TOUCH_BUTTONS)
+
+    //
+    // Screen Click
+    //  - On menu screens move directly to the touched item
+    //  - On select screens (and others) touch the Right Half for +, Left Half for -
+    //
+    void MarlinUI::screen_click(const uint8_t row, const uint8_t col, const uint8_t x, const uint8_t y) {
+      if (screen_items > 0)
+        encoderDiff = ENCODER_PULSES_PER_STEP * (row - encoderPosition + encoderTopLine);
+      else
```

Comment:
i would replace this else by "else if (!on_status_screen()) to avoid feedrate change by mistake.... or/and maybe try to trigger icon's clicks ...